recipes/redradist-icc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION ${CMAKE_VERSION})
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
It is bad idea, I added `${CMAKE_VERSION}` exactly, because CI failed with wrong CMAKE_VERSION ...
Why do you want me hard-code the version ?

Reply:
CMake does this automatically for us =)